core/opendaq/reader/tests/test_multi_reader.cpp

```diff
@@ -1813,4 +1813,68 @@ TEST_F(MultiReaderTest, StartOnFullUnitOfDomain)
 
     ASSERT_THAT(time[1], ElementsAreArray(time[0]));
     ASSERT_THAT(time[2], ElementsAreArray(time[0]));
-}
\ No newline at end of file
+}
+
+TEST_F(MultiReaderTest, SampleRateDivider)
```

Comment:
A test of an expected error when the signals do not have an integer divider would be a good addition.